### ISSUE TITLE
The Hackage-released `hackage-server` has no build plan beyond GHC 7.8

### ISSUE THREAD

**Issue by andreasabel** [haskell/hackage-server#1129]
I tried to build the code obtained via `cabal get hackage-server`.  It has no build plan for GHC 7.10, 8.x, 9.x.

Maybe time to release `hackage-server` to hackage at some point?
E.g. why doesn't the deployed hackage server correspond to the one released on hackage?

**Comment by gbaz** [comment]
I agree -- we've gone a long time without a hackage release for no particular reason. I'm open to making one at some reasonable point.
